src/server/lib/__tests__/updateRegistrationLocation.test.ts

```diff
@@ -26,14 +26,15 @@ jest.mock('@/server/lib/serverSideLogger', () => ({
 const getFakeRequest = (
 	country: string | undefined,
 	consent: string | undefined,
-) => ({
-	cookies: {
-		GU_geo_country: country,
-	},
-	body: {
-		_cmpConsentedState: consent,
-	},
-});
+): Request =>
+	({
+		cookies: {
+			GU_geo_country: country,
+		},
+		body: {
+			_cmpConsentedState: consent,
+		},
+	}) as unknown as Request;
```

Comment:
not sure why this started failing when running unit tests, but using `as` syntax is fine here as its for a test